### ISSUE TITLE
Client is outdated

### ISSUE THREAD

**Issue by lwindolf** [grafeas/client-ruby#1]
Latest API provided by code in grafeas uses "v1beta" API, so this client needs to be regenerated.

**Comment by aysylu** [comment]
@lwindolf if you'd like to regenerate it and open PR, we welcome contributions!

**Comment by lwindolf** [comment]
I've seen several people try in the closed PRs. Given the problems with the broken swagger API JSON (POST URLs with wrong definitions) regeneration is IMO not possible. This is something only the Grafeas project can fix.

**Comment by aysylu** [comment]
Yes, the Swagger API <-> proto bindings incompatibility is the reason these can't easily be auto-generated. There are 3 possible paths here:

1) auto-generate and then fix the issues (e.g. https://github.com/grafeas/client-go/pull/15),
2) generate brand new client with the client methods implemented idiomatically,
3) use protoc-generated code.

If you'd like to contribute 1) or 2), happy to review!

**Comment by lwindolf** [comment]
Sorry the CLA prevents me from contributing. Also I don't like the unapproachability of the project. Asking for potential users to fix what they need to approach the project in the first place is IMO not ok if this is to be considered an OpenSource project looking for a community.

**Comment by lwindolf** [comment]
FYI I blog posted on how to workaround the issue: https://lzone.de/blog/Accessing-Grafeas-with-a-swagger-client. Maybe this helps others.

**Comment by aysylu** [comment]
Adding current set of maintainers @wkozlik @nyc for visibility, as I'm no longer maintaining this project.

**Comment by nyc** [comment]
I believe the issue that was mentioned previously should be fixed with https://github.com/grafeas/grafeas/pull/532 so we can regenerate the client now. I'm also working on merging the Swagger files in v1beta1 so you don't have to generate two separate clients for the `grafeas` and `projects` services and eventually generating them in v1 so we can generate clients for all of the versions.
